### PR TITLE
feat: use latest d3 version from npm

### PR DESCRIPTION
**Pull request by chmelevskij**
* use npm `d3` version
* update `d3` usages to new api

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]
* successfully flashed 1 unsoldered FC's with 4.3.1 and zulu-cloud; 
* minimal configs of 2 unsoldered FC
* motors and sensors tab graphs worked fine on both FC; i could not test receiver tab without RX.

**Pull request merged**
